settings.local.generic.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @file
+ * Local Backdrop CMS configuration file.
+ */
+
+/**
+ * Simple database settings:
```

Comment:
This is no longer an example of simple database settings.  Simple is the single line:

`$database = 'mysql://username:password@127.0.0.1/backdropcmsorg';`	 

We're using "the advanced database settings" array instead.

Reply:
Now including both.